user_guide/data_management.rst

```diff
@@ -0,0 +1,234 @@
+
+
+How the data is organized
+-------------------------
+
+The customer relationship management process requires us to collect and process huge amount of information, and OroCRM 
+is sharpened to do it in the most efficient and convenient way.
+
+Technically, a collection of information of a similar nature is represented in OroCRM with an **Entity**. Each instance 
+of this collection is called a **Record**. Details of each record are its **Properties**.
+
+For example, information about customers of a Web-store is collected as properties of the records of the Web Customer 
+entity. This means, that one of the OroCRM entities is “Web Customer” and the system knows that records of this 
+entity can have a number of properties, such as the first name, last name, email, phone number, id of the shipping 
+address, id of the cart and ids of the orders. Moreover, the system is aware that some of this properties must be 
+defined and some are optional. When a new Web customer is added to the system – its record is created and values of 
+the properties are save for it. If something has changed (e.g. the phone number), the properties are updated. 
+
+New records can be created and the existing one can be updated both manually and automatically (as a result of 
+integration with third-party systems). 
+With OroCRM's comprehensive :ref:`access settings <user-guide-user-management-permissions>` you can define what users 
+will be able to view, update, delete, and create records of specific entities.
+
+As soon as the property has been updated, all the system users who are allowed to 
+see the property, will see its new updated state. This ensures that all the stakeholders can obtain actual and 
+up-to-date information at any moment. 
+
+
+How the Entities Can Be Related
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Some pieces of information are complex enough to have their own entity. For example, for each cart in a Web store, we 
+need to know its unique id (number), how many items are in the cart, what their total value is, whether a purchase has 
+been made. Therefore, we have an entity “Cart”, records of which have all these properties: number of items, total 
+value, status. 
+
+One customer may have several carts. For OroCRM this means that Web Customer records are related to Carts 
+records as one to many. To show this, we assign one of the Web Customer properties as a “relation” to carts. This 
+field will save identificators of the customer’s carts (such identificator is a field of the cart). By the 
+identificator the system can find the cart and bind its properties to the customer. 
+
+Moreover, each cart may have several 
+items in it. This means that each Cart record is related to its Item records as one to many. 
+
+*User-wise, this means that you can get easily see the full picture and drill-down into details as much as required for 
+any specific task, as well as update the process the provided data.*  
```

Comment:
Edit: *User-wise, this means that you can easily get a full picture and drill-down into details as much as required for any specific task, as well as update the process with the provided data.*